testutil/test_helper.go

```diff
@@ -336,6 +336,16 @@ func (h *TestHelper) LinkerdRun(arg ...string) (string, string, error) {
 	return h.PipeToLinkerdRun("", arg...)
 }
 
+// LinkerdRunOk executes a linkerd command appended with the --linkerd-namespace
+// flag.
+func (h *TestHelper) LinkerdRunOk(arg ...string) (string, error) {
+	out, stderr, err := h.PipeToLinkerdRun("", arg...)
+	if err != nil {
```

Comment:
I like this extra context on the error.  Can we just do this in `LinkerdRun`?  That way I don't think we need to add `LinkerdRunOk`.  Callers who don't specifically care about the stderr can safely drop the second return value since the stderr will also be present in the error if there is an error.

Reply:
There are one or two cases where we actually want to look at error messages

Reply:
I've renamed `LinkerdRunOk` to `LinkerdRun` and changed all other uses to PipeToLinkerdRun